src/pages/404.f7.jsx

```diff
@@ -0,0 +1,28 @@
+export default (AB) => {
+   return (props, { $, $h, $f7, $on, $store, $update }) => {
```

Comment:
Pretty minor, but since we don't actually read any of the params, get rid of them?
```js
export default () => {
   return () => {
```